src/index.js

```diff
@@ -59,5 +92,6 @@ const css = (...styleDefinitions) => {
 export default {
```

Comment:
I just noticed this. I'm afraid to say that this is using ES6 modules incorrectly and it'll bite us when this project is upgraded to Babel 6. Learn more about my painful learning experience here: http://kcd.im/tears